libs/template/resources/src/graphics/common/logo.ts

```diff
@@ -1,5 +1,11 @@
-const symbol = `
-<g part="symbol">
+import { featureVersion } from '@spryker-oryx/utilities';
+
+const darkMode =
+  featureVersion >= '1.4'
```

Comment:
In case we are relying on the `1.4` version then it's a `feat` probably. But it looks like a `fix` to me, so then we probably should backport it, and rely on `1.3` version.